### ISSUE TITLE
`sver validate` does not detect default profile missing.

### ISSUE THREAD

**Issue by kna** [mitoma/sver#116]
It the default profile is not on the `sver.toml` and you refer it, `sver calc` fails.

Example:
```
.
├── data1
│   └── sver.toml (1)
└── data2
     └── sver.toml (2)


# sver.toml(1)
[default]
dependencies = [
  "data2",
]

# sver.toml(2)
[non-default]
dependencies = [
   ...
]
```

Commands:
```
% sver --version
sver 0.1.14
% sver validate     
[OK]	data1/sver.toml:[default]
[OK]	data2/sver.toml:[no-default]
% sver calc data1    
profile[default] is not found
```

It seems correct that `sver calc` raises an error.
`sver validate` should detect it.

**Comment by mitoma** [comment]
@kna 
Thank you for the report. This behavior is a bug and has been fixed in #122.

**Comment by kna** [comment]
Thank you for fixing!
